From c303c5237679259f232b81c955c85e03d3be09e8 Mon Sep 17 00:00:00 2001
From: Marco Meyer-Conde <marco@tcu.ac.jp>
Date: Thu, 17 Jul 2025 14:52:55 +0900
Subject: [PATCH] yamlcpp-stdint

---
 src/yamlcpp/emitterutils.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/yamlcpp/emitterutils.h b/src/yamlcpp/emitterutils.h
index 93ad79e..b33c955 100644
--- a/src/yamlcpp/emitterutils.h
+++ b/src/yamlcpp/emitterutils.h
@@ -8,6 +8,7 @@
 #endif
 
 #include <string>
+#include <cstdint>'
 
 #include "emitterstate.h"
 #include "yaml-cpp/emittermanip.h"
-- 
2.34.1

